languages/python/sqlalchemy-oso/tests/test_sqlalchemy.py

```diff
@@ -426,20 +406,64 @@ def test_bakery_caching_for_scoped_session(engine, oso, fixture_data):
     # permitting access to "read" users.
     assert first_authorized_post.created_by is None
 
-    # Explicitly pass `enable_baked_queries=True`.
-    baked_authorized_session = scoped_session(
-        lambda: oso, lambda: "user", lambda: "read", enable_baked_queries=True
-    )
-    baked_authorized_session.configure(bind=engine)
-    baked_authorized_posts = baked_authorized_session.query(Post)
-    assert baked_authorized_posts.count() == 1
-    first_baked_authorized_post = baked_authorized_posts[0]
-    assert first_post.id == first_baked_authorized_post.id
-
-    # Should be able to view the post's creator because it's using the User
-    # query baked by `first_post.created_by`.
-    #
-    # NOTE(gj): This is actually an authorization bug and not desired behavior,
-    # but we're testing that folks are able to use baked queries if they
-    # understand the risks and explicitly pass `enable_baked_queries=True`.
-    assert first_baked_authorized_post.created_by.id == first_post.created_by.id
+
+def test_register_models_declarative_base():
+    """Test that `register_models()` registers models."""
+    from oso import Oso
+    from polar.exceptions import DuplicateClassAliasError
+
+    from sqlalchemy_oso.auth import register_models
+
+    from .models import Category, ModelBase, Tag
+
+    oso = Oso()
+    register_models(oso, ModelBase)
+
+    for m in [Category, Post, Tag, User]:
+        with pytest.raises(DuplicateClassAliasError):
+            oso.register_class(m)
+
+
+if AT_LEAST_SQLALCHEMY_VERSION_1_4:
```

Comment:
I think the idiomatic way to do this is to skip the test: https://docs.pytest.org/en/6.2.x/skipping.html#id1

Reply:
efef45c